packages/common/src/scopeSupportFacets/html.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  LanguageScopeSupportFacetMap,
+  ScopeSupportFacetLevel,
+} from "@cursorless/common";
+
+const { supported, notApplicable } = ScopeSupportFacetLevel;
+
+export const htmlSupport: LanguageScopeSupportFacetMap = {
+  ["key.attribute"]: supported,
+  ["tags.element"]: supported,
```

Comment:
Why the `.element` here?

Reply:
We might have tags in another context in the future. I'm just trying to be thorough.

Reply:
idk either

how immutable do we need these facets to be? can we lower the cost of mistakes sufficiently that it doesn't matter much?

Reply:
Were gonna have a lot of files with these facets as their name so it's not gonna be that easy to change them, but also definitely not impossible.

Reply:
Update from meet-up: let's go with just `tags`. It's ok to have the general thing and then sub-facets as well